game/src/edit/roads.rs

```diff
@@ -757,27 +773,34 @@ fn make_main_panel(
             .text("Access restrictions")
             .build_def(ctx)
             .centered_vert(),
-    ]);
+    ])
+    .section(ctx);
 
-    Panel::new_builder(Widget::col(vec![
-        modify_lane,
-        available_lane_types_row,
+    Panel::new_builder(Widget::custom_col(vec![
+        road_settings,
+        add_lane_row.margin_below(16),
```

Comment:
Any reason to not add the same margin below the lane cards, for that consistent "floating" look? A preview:

![Screenshot from 2021-08-25 09-53-30](https://user-images.githubusercontent.com/1664407/130832649-27fae668-7943-446b-9736-fae7175301e6.png)